clients/runtime/src/types.rs

```diff
@@ -246,7 +251,7 @@ mod dispatch_error {
 		Unsupported,
 	);
 
-	convert_enum!(RichArithmeticError, ArithmeticError, Underflow, Overflow, DivisionByZero,);
+	// convert_enum!(RichArithmeticError, ArithmeticError, Underflow, Overflow, DivisionByZero,);
```

Comment:
Why is this commented out?

Reply:
fixed.
reuse auto implement macros instead of match statement for ArithmeticError